server/src/test/java/keywhiz/service/crypto/CryptoModuleTest.java

```diff
@@ -0,0 +1,39 @@
+package keywhiz.service.crypto;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import java.security.SecureRandom;
+import keywhiz.KeywhizConfig;
+import keywhiz.inject.StrictGuiceModule;
+import keywhiz.test.ServiceContext;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNotSame;
+
+public class CryptoModuleTest {
+  @Test
+  public void secureRandomIsExplicitlyBound() {
+    KeywhizConfig config = ServiceContext.create().getConfig();
+
+    Injector injector = Guice.createInjector(
+        new CryptoModule(
+            config.getDerivationProviderClass(),
+            config.getContentKeyStore()),
+        new StrictGuiceModule());
```

Comment:
How did you know to include `StrictGuiceModule` here?

Reply:
I needed to verify that, with the restrictions turned on, we could still inject instances of SecureRandom. Basically, I wanted the Injector in this test to faithfully reproduce the behavior of the service's actual Injector.